tests/basic/distribute/dht_seek_test.t

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+. $(dirname $0)/../../include.rc
+. $(dirname $0)/../../volume.rc
+. $(dirname $0)/../../dht.rc
+
+SCRIPT_TIMEOUT=300
```

Comment:
Does this test really require more than 200 seconds (the default value) to complete ?

Reply:
done